hpx/components/containers/partitioned_vector/partitioned_vector.hpp

```diff
@@ -337,6 +337,15 @@ namespace hpx
             register_as(symbolic_name).get();
         }
 
+        // construct from id
+        partitioned_vector(future<id_type> && f)
+        {
+            using util::placeholders::_1;
+            f.share().then(
```

Comment:
`partitioned_vector::connect_to_helper` expects to be called with a shared_future.